daangn.css

```
@@ -0,0 +1,161 @@
+/* footer body */
+#footer_root {
+  border-top: 1px solid lightgray;
+  padding: 0 1.6rem;
+  box-sizing: border-box;
+  width: 100%;
+  max-width: 100vw;
+}
+.footer_body {
+  display: flex;
+  flex-direction: column;
+  max-width: 50rem;
+  margin: 0 auto;
+}
+
+/* footer section 1 */
+.footer_section1 {
+  padding: 4.8rem 0;
+  display: block;
+}
+.footer_menus {
+  display: flex;
+  max-width: 76.8rem;
+  font-size: 0.8rem;
```

폰트 크기가 아주 조금 더 커도 좋을 것 같아요...!!
```suggestion
  font-size: 0.9rem;
```